include/sir/platform.h

```diff
@@ -202,6 +202,7 @@ int pthread_getname_np(pthread_t thread, char* buffer, size_t length);
 #  endif
 # else /* _WIN32 */
 #  define __WIN__
+#  undef SIR_NO_SYSTEM_LOGGERS
```

Comment:
This reminds me... when I was running the test rig earlier, SIR_OS_LOG_ENABLED was not defined, so I think something broke at some point. I will take a look.

Reply:
We should definitely add it to the CI pipeline, if unable to do so in Docker, I can set up a VM on my iMacPro and we can figure out how to run it there